client/controller/index.node.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+import { setSection as setSectionAction } from 'state/ui/actions';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import LayoutLoggedOut from 'layout/logged-out';
+
+export function makeLayoutMiddleware( LayoutComponent ) {
+	return ( context, next ) => {
+		const { store, primary, secondary, tertiary } = context;
+
+		context.layout = (
+			<LayoutComponent store={ store }
+				primary={ primary }
+				secondary={ secondary }
+				tertiary={ tertiary }
```

Comment:
I still see it being used in https://github.com/Automattic/wp-calypso/blob/master/client/reader/controller.js...